File: solutions/java15/medium/class-and-objects-sports-event-venue-management-system-1/solution-approach1.java
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/contests/ku-uit-oops/challenges/class-and-objects-sports-event-venue-management-system-1/problem?isFullScreen=true
// Problem     Class and Objects - Sports Event Venue Management System 1
// Difficulty  Medium
// Subdomain   N/A
// Platform    HackerRank
// Language    java15
// Status      Accepted
// Submitted   2026-08-19, 12:19 p.m.
// ──────────────────────────────────────────────────

import java.io.*;
import java.util.*;

class Venue{
    private String venue;
    private String city;
    
    Venue(String venue, String city){
        this.venue=venue;
        this.city=city;
    }
    void display(){
        System.out.println("Venue Details");
        System.out.println("Venue Name : "+this.venue);
        System.out.println("City Name : "+this.city);
        System.out.println(this.venue);
    }
}
public class Solution {

    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the venue name");
        String venue=sc.nextLine();
        System.out.println("Enter the city name");
        String city=sc.nextLine();
        Venue v=new Venue(venue, city);
        v.display();
        sc.close();
    }
    
}

File: solutions/cpp/medium/input-output-student-information-management-system-3/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/contests/ku-uit-dsa-beginner/challenges/input-output-student-information-management-system-3/problem?isFullScreen=true
// Problem     Input & Output - Student Information Management System
// Difficulty  Medium
// Subdomain   N/A
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-08-21, 09:07 p.m.
// ──────────────────────────────────────────────────

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <iomanip>
#include <string>

using namespace std;

int main() {
    string name;
    int age;
    float cgpa;
    char grade;

    cin >> name;
    cin >> age;
    cin >> cgpa;
    cin >> grade;
    cgpa = ((int)(cgpa * 100)) / 100.0f;

    cout << "Name: " << name << endl;
    cout << "Age: " << age << endl;
    cout << "CGPA: " << fixed << setprecision(2) << cgpa << endl;
    cout << "Grade: " << grade << endl;

    return 0;
}
